experimental/listener.go

```diff
@@ -24,6 +24,16 @@ type StackIterator interface {
 	Parameters() []uint64
 }
 
+// Globals is a proxy to the set of globals available to the context of a
+// function call.  This is useful for inspecting globals, but not modifying
+// them.
+type Globals interface {
+	// Count returns the number of globals registered at this moment.
+	Count() int
```

Comment:
the count of globals is static for the life of the module isn't it? If so, "registered at this moment" may be misleading.

Are you adding this here because the data isn't available on CompiledModule?

Reply:
> the count of globals is static for the life of the module isn't it?

Didn't realize that, thank you!

> Are you adding this here because the data isn't available on CompiledModule?

Added it because `api.Module` doesn't provide that information (only for exported globals).